src/content/ui-api/management-api/authentication/index.md

```diff
@@ -86,6 +86,43 @@ This page will also present your Management API endpoint's certificate authority
 
 The rest of the page helps you set up `kubectl` manually, adaptable for various operating systems.
 
+## Web UI login {#web-ui}
+
+Our Web UI provides a simple single sign-on mechanism that will simply send a user through your identity provider's authentication process and finally redirect to the web UI. Behind the scenes, the same mechanism is used as in the examples above.
+
+## Details of an ID token {#id-token-details}
+
+Regardless of which login method is used, once this is done, the user's client (web UI, kubectl etc.) will send an ID token with every request to the Management API. The ID token contains information about the user which can then be used in [authorization]({{< relref "/ui-api/management-api/authorization/index.md" >}}) to decide which permissions the user should be granted.
+
+The ID token is a [JSON Web Token](https://datatracker.ietf.org/doc/html/rfc7519) (JWT). The payload part of an ID token issued for the Management API can look like in the following example. Note that we omit some parts that are not relevant for the purpose of this article.
+
+```json
+{
+  "iss": "https://dex.g8s.garlic.eu-west-1.aws.gigantic.io",
+  "exp": 1645533923,
+  "iat": 1645532123,
+  "email": "jane@example.com",
+  "groups": [
+    "customer:GiantSwarmAdmins"
+  ],
+  "name": "Jane Smith",
+  "preferred_username": "janes",
+  ...
+}
+```
+
+Let's go into details for the most relevant properties (also called "claims") of this payload object.
+
+| Claim | Description |
+|-|-|
+| `iss` | The OIDC provider that has issued the token. In our case it is Dex running in the management cluster. The Management API will only|
```

Comment:
```suggestion
| `iss` | The OIDC provider that has issued the token. In our case it is Dex running in the management cluster.|
```
There's a start of a sentence here - could we remove it or complete it?

Reply:
Remove :)